engines/wintermute/ad/ad_scene.cpp

```diff
@@ -1116,16 +1116,14 @@ bool AdScene::display() {
 //////////////////////////////////////////////////////////////////////////
 bool AdScene::updateFreeObjects() {
 	AdGame *adGame = (AdGame *)_gameRef;
-	// 3D-code removed
-	// bool is3DSet;
+	bool is3DSet = false;
```

Comment:
This would produce a warning about unused variable, no?

Reply:
Well, yes. But @aquadran and @GunnarBirke were going to use this variable anyway.

Reply:
Discussed with @aquadran, removing this line.